tests/agent/test_event_handler.py

```diff
@@ -40,21 +39,32 @@
     RunOnboardingDataAssistantEvent,
     UnknownEvent,
 )
+from great_expectations_cloud.agent.warnings import GXAgentUserWarning
 
 if TYPE_CHECKING:
     from pytest_mock import MockerFixture
 
 pytestmark = pytest.mark.unit
 
 
+@pytest.fixture()
+def example_event():
+    return RunOnboardingDataAssistantEvent(
+        type="onboarding_data_assistant_request.received",
+        datasource_name="abc",
+        data_asset_name="boo",
+    )
+
+
 class TestEventHandler:
-    def test_event_handler_raises_for_unknown_event(self, mock_context):
+    def test_event_handler_unknown_event(self, mocker):
         event = UnknownEvent()
         correlation_id = "74842258-803a-48ca-8921-eaf2802c14e2"
-        handler = EventHandler(context=mock_context)
-
-        with pytest.raises(UnknownEventError):
-            handler.handle_event(event=event, id=correlation_id)
+        context = mocker.MagicMock(autospec=CloudDataContext)
```

Comment:
Not a blocker, but you can alternatively use the `mock_context` fixture in this test.

Reply:
updated in 04c5f68